### ISSUE TITLE
Screen reader not announcing expand/collapse state for expand/collapse toggle in grouped list 

### ISSUE THREAD

**Issue by brism17** [microsoft/fluentui#17263]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

This is similar to this issue: https://github.com/microsoft/fluentui/issues/17203 - screen reader not announcing expand/collapsed state in detailed grouped list. 
For the "Expand/collapse" button , the NVDA only announces the button name  but there is no announcement of the state of the controls whether the control is in expanded or collapsed state

### Please provide a reproduction of the issue in a codepen:
1. open a basic example of grouped list in codepen . 
2. Start the screen reader 
3. Notice the screen reader doesn't say anything about the current state of the expanded /clsoed button. 

#### Actual behavior:

For the "Expand/collapse" button , the NVDA only announces the button name  but there is no announcement of the state of the controls whether the control is in expanded or collapsed state
#### Expected behavior:

For the "Expand/collapse" button , the NVDA should announce the  the state of the control whether the control is in expanded or collapsed and also NVDA should announce the state after activating the control, means if we are changing the state from Collapsed to Expand then NVDA should announce the updated state.

### Documentation describing expected behavior
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d

**Comment by gouttierre** [comment]
@brism17 - Thank you for filling this issue with us.
 
@khmakoto - Would you be able to verify if this issue is related to #17203? 
cc @smhigley

**Comment by miduncan** [comment]
@gouttierre @khmakoto Ping on this

**Comment by smhigley** [comment]
I can't reproduce this bug -- NVDA announces the expanded/collapsed state for me in Firefox and Chromium in both browse mode and forms mode. This works both when I first arrive on the control, and when I change it, for both individual rows and the header cell. I'm checking on the [simple grouped DetailsList](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/grouped).
